overview/style.css

```diff
@@ -286,6 +286,42 @@ code, pre {
   overflow-x: auto;
 }
 
+.code-block {
+  position: relative;
+}
+
+.code-buttons {
+  position: absolute;
+  top: 0.5em;
+  right: 1em;
+}
+
+.button {
+  display: inline-block;
+  padding: 0.1em 0.4em;
+  background-color: transparent;
+  color: white;
+  text-align: center;
+  text-decoration: none;
+  border-radius: 5px;
+  border: solid 1px gray;
+  background-color: gray;
+  margin: 0;
+  font-size: 0.8rem;
+}
+
+.button.download {
```

Comment:
Can we have a download and copy symbol in front of the text please?

Reply:
Tried my best, even though they don't look great

![tmp xzyezad29B](https://github.com/user-attachments/assets/72232d0c-d2a3-4ca5-99b3-5b3b0ec4acd2)

Reply:
That's okay, the icons are font-specific and we can still fix the font later

Reply:
Oh yes that symbol is way better

![tmp NePBMAxo9V](https://github.com/user-attachments/assets/730ab0a3-5c1c-4c0e-966b-5471f3c68184)

Reply:
I'd say that icons would be descriptive enough on their own. It's pretty universally known what the download and copy symbols look like.

Reply:
@eljamm I disagree, apart from that there's no such thing as universality, the well-known-ness depends on the specific audience. Shift by a couple of years of age up and people won't even know what a download is, shift a couple of years down and they may not know what a file is. Using multiple representations is good.

Reply:
Okay, but can we have a bigger icon, at least? It doesn't really look that visible to me (tried on Firefox and Chrome):

![image](https://github.com/user-attachments/assets/063a0a0a-70c1-4ace-8532-38b39d4b82a2)